Office 365/o365/_meta/fields.yml

```diff
@@ -60,6 +60,26 @@ email.to.address:
   name: email.to.address
   type: keyword
 
+office365.browser.name:
+  description: ""
+  name: office365.browser.name
+  type: keyword
+
+office365.browser.version:
+  description: ""
+  name: office365.browser.version
+  type: keyword
+
```

Comment:
My bad. These two fields can be replaced with `user_agent.*`:
```suggestion
```

Reply:
Ok, yes, I fix that